app/components/dashboard/quick-update/item.js

```diff
@@ -31,9 +31,11 @@ export default Component.extend({
     if (get(this, 'isCompleted')) {
       return get(this, 'intl').t('dashboard.quick-update.media.complete');
     }
-    const type = get(this, 'entry.media.modelType');
-    const num = get(this, 'nextProgress');
-    const text = get(this, 'intl').t('dashboard.quick-update.media.episode', { type, num });
+    const text = get(this, 'intl').t('dashboard.quick-update.media.episode', {
+      type: get(this, 'entry.media.modelType'),
+      num: get(this, 'nextProgress'),
```

Comment:
imo is kinda confusing
![](https://puu.sh/v8wRA/d23915f5e7.png)
It makes it look like I've already watched the 2nd episode and are about to mark the 2nd episode as complete.